tasks/post-to-cmr/tests/cmr_test.js

```diff
@@ -80,6 +80,36 @@ test.serial('should succeed with correct payload', (t) => {
     }));
 });
 
+test.serial('should succeed with correct payload and return SIT url', (t) => {
+  process.env.CMR_ENVIRONMENT = 'SIT';
+  const newPayload = JSON.parse(JSON.stringify(payload));
+  sinon.stub(cmrjs.CMR.prototype, 'ingestGranule').callsFake(() => ({
+    result
+  }));
+  const granuleId = newPayload.input.granules[0].granuleId;
+  const key = `${granuleId}.cmr.xml`;
+
+  return aws.promiseS3Upload({
+    Bucket: t.context.bucket,
+    Key: key,
+    Body: fs.createReadStream('tests/data/meta.xml')
+  }).then(() => postToCMR(newPayload)
+    .then((output) => {
+      cmrjs.CMR.prototype.ingestGranule.restore();
+      delete process.env.CMR_ENVIRONMENT;
+      t.is(
+        output.granules[0].cmrLink,
+        `https://cmr.sit.earthdata.nasa.gov/search/granules.json?concept_id=${result['concept-id']}`
+      );
+    })
+    .catch((e) => {
```

Comment:
With a `.finally()` block, you could actually get rid of this catch block and let ava detect the test failure when the exception is thrown.

Reply:
The reason for the catch I believe to be the call to `cmrjs.CMR.prototype.ingestGranule.restore();`, doesn't that necessitate leaving it in? I could change it to `throw e` instead of calling `fail()`